lib/puppet/provider/sensu_user/sensu_api.rb

```diff
@@ -74,6 +74,7 @@ def create
         configure_cmd << '--trusted-ca-file'
         configure_cmd << resource[:configure_trusted_ca_file]
       end
+      Puppet.notice('Executing sensuctl configure')
```

Comment:
Are these present in a puppet run? Could you paste an example?

Reply:
This is from vagrant where I changed admin password:

```
[root@sensu-backend ~]# puppet apply backend.pp 
Notice: Compiled catalog for sensu-backend in environment production in 1.16 seconds
Notice: /Stage[main]/Sensu::Backend/Sensu_user[admin]/password: changed password
Notice: Executing sensuctl configure
Notice: Applied catalog in 4.97 seconds
```